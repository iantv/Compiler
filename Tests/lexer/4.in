"asdasdasd" 'asdasdas'
"ad"asda"sda" 'asd'asd'asd'