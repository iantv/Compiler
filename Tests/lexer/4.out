1	1	1.123456789		dbl_val
